Tiltfile

```diff
@@ -454,6 +480,82 @@ def waitforsystem():
     local(kubectl_cmd + " wait --for=condition=ready --timeout=300s pod --all -n capi-kubeadm-control-plane-system")
     local(kubectl_cmd + " wait --for=condition=ready --timeout=300s pod --all -n capi-system")
 
+def peer_vnets():
+    # TODO: check for az cli to be installed in local
+    # wait for AKS VNet to be in the state created
+    peering_cmd = '''
+    ; echo \"--------Peering VNETs--------\""
```

Comment:
Are we trying to print the quotation marks? If so, this is correct, but I wasn't sure why they would be included. Quotes don't need to be escaped generally in multiline string literals. I thought the output
```
--------Peering VNETs--------
```
was sufficient, but I may be confused.